devenv.md

```diff
@@ -0,0 +1,258 @@
+  In order to start the setup procedure, please follow all steps in section [1 - Prerequisites](#prereq) 
+since every step in the following section [*2*](#repositories) depends on it. Then you can proceed and
+set up each required project. This should be again straigt forward process, as it requires following
+each step mentioned in the project section.
+
+  Because we realize that this is quite complicated and rather tedious process, we prepared a [script](#bscript)
+and a set of pre-prepared [images](#vagrant). 
+
+## <a name="prereq"></a>1. Prerequisites
+
+**Step 1) Enabling repositories containing build & runtime libraries:**
+
+```shell
+    dnf copr enable evilissimo/leapp
+```
+
+**Step 2) Installing required tools:**
+
+```shell
+    dnf -y install git tito python2-pip
+```
+
+**Step 3) Setting up a project root**
+```shell
+    export LEAPP_HOME=$HOME/leapp
+    mkdir -p $LEAPP_HOME
+```
+
+
+## <a name="repositories"></a>2. LeApp repositories
+
+### <a name="snactor"></a>2.1 Snactor library
+Snactor library is a runtime dependency of LeApp daemon. This library is responsible for
+executing and handling of so-called actors.
+
+#### Step 1) Repository cloning \*
+
+```shell
+    git clone https://github.com/leapp-to/snactor.git $LEAPP_HOME/snactor
+```
+
+\* *Fork the repository if you want to open pull requests in the future*
+
+#### Step 2) External dependencies & packages
+
+```shell
+    dnf -y install ansible
+```
+
+#### Step 3) Build & installation:
+
+```shell
+    cd $LEAPP_HOME/snactor
+    dnf builddep python-snactor.spec
+    tito build -i --verbose --rpm --test
+```
+
+#### Step 4) Setting up a testing environment - unit testing:
+
+```shell
+    cd $LEAPP_HOME/snactor
+    pip install --user -r requirements.txt -r requirements-test.txt
+```
+
+*Test execution example:*
+
+```shell
+    make test
+```
+
+---
+
+### <a name="daemon"></a>2.2 LeApp-daemon
+LeApp daemon is writen in go-lang therefore an working golang environment is
+required to build & run it. If you do not have any, please follow these basic steps
+in order to setup one.
+
+```shell
+    dnf -y install go
+    mkdir -p $HOME/go/src/github.com
+    echo "export GOPATH=$HOME/go" >> $HOME/.bashrc
+    exec bash
+```
+
+Shall you need more information, kindly visit [gostart](https://github.com/alco/gostart)
+
+#### Step 1) Repository cloning \*
+
+```shell
+    mkdir -p $GOPATH/src/github.com/leapp-to
+    git clone https://github.com/leapp-to/leapp-go.git $GOPATH/src/github.com/leapp-to/leapp-go
+    ln -s $GOPATH/src/github.com/leapp-to/leapp-go $LEAPP_HOME/leapp-go
+```
+
+\* *Fork the repository if you want to open pull requests in the future*
+
+
+#### Step 2) Build & installation:
+
+**Requires**
+ + [Snactor library](#snactor)
+
+```shell
+    go get github.com/gorilla/mux
+    cd $LEAPP_HOME/leapp-go
+    make build
+```
+
+#### Step 3) Setting up a testing environment - unit testing:
+
+```shell
+    dnf -y install golang-googlecode-tools-goimports
```

Comment:
this should be before ```make build``` in line 116

Reply:
Yeah, it is in the script with pkg/error, but somehow I didn't copy it there.